merino/providers/adm/provider.py

```diff
@@ -123,9 +123,13 @@ async def _fetch(self) -> None:
     def hidden(self) -> bool:  # noqa: D102
         return False
 
+    def normalize_query(self, query: str) -> str:
+        """Convert a query string to lowercase and remove trailing spaces."""
+        return query.strip().lower()
+
     async def query(self, srequest: SuggestionRequest) -> list[BaseSuggestion]:
         """Provide suggestion for a given query."""
-        q = srequest.query
+        q: str = self.normalize_query(srequest.query)
```

Comment:
nitpick: Since you're normalizing the query in `api_v1` now, you can revert this change!

```suggestion
        q = srequest.query
```

Reply:
doh! Thank you!